server/wfnews-api-rest-endpoints/src/main/resources/application-secrets.properties

```diff
@@ -1,2 +1 @@
-wfone.datasource.password=#{wfone.datasource.password}
-
+wfone.datasource.password=Rf%#yZVQQ%Y4tPk6
```

Comment:
NEVER hard code passwords. Use a config.
If you're doing things locally, that's fine, but DO NOT COMMIT SECURE DATA